packages/tasit-action/src/Subscription.js

```diff
@@ -84,18 +82,28 @@ export class Subscription {
 
     const message = { error };
     errorEventListener.listener(message);
-  }
+  };
 
   // TODO: Make protected
-  // Arrow functions in class properties won't be in the prototype and we can't call them with super
-  // Refs: https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1
-  _emitErrorEventFromEventListener(error, eventName) {
+  _emitErrorEventFromEventListener = (error, eventName) => {
     error.eventName = eventName;
     this._emitErrorEvent(error);
-  }
+  };
 
   // TODO: Make protected
-  _addErrorListener = listener => {
+  _addErrorListener = newListener => {
+    let listener = newListener;
+    const errorEventListener = this.#eventListeners.get("error");
+
+    if (errorEventListener) {
+      listener = error => {
+        newListener(error);
+        errorEventListener.listener(error);
```

Comment:
Oh I get it, is this a hack to allow for two listeners, the catch-call one that the contract added and then the custom one?

Reply:
Yes, it's necessary because we are allowing only one listener per event (error included).

Reply:
Hmm yeah. I'll have to think about this more, probably as part of this thread https://github.com/tasitlabs/TasitSDK/issues/164

Should we extract it there?

Reply:
Done.

Reply:
Thanks. Resolving this